app/models/event_date.rb

```diff
@@ -1,7 +1,7 @@
 class EventDate
   attr_reader :t, :secs
   def initialize
-    @t = Time.zone.now
+    @t = Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.